src/plugins/vis_types/timeseries/public/application/components/lib/index_pattern_select/index_pattern_select.tsx

```diff
@@ -34,9 +34,9 @@ export interface IndexPatternSelectProps {
 }
 
 const defaultIndexPatternHelpText = i18n.translate(
-  'visTypeTimeseries.indexPatternSelect.defaultIndexPatternText',
+  'visTypeTimeseries.indexPatternSelect.defaultDataViewText',
   {
-    defaultMessage: 'Default index pattern is used.',
+    defaultMessage: 'Default data view is used.',
```

Comment:
This wording seems awkward to me. I'd change it to:

```suggestion
    defaultMessage: 'Using the default data view.',
```

There's also a minor spacing issue if you disable data views while the default data view is selected.

<img width="423" alt="Screen Shot 2021-10-01 at 2 15 53 PM" src="https://user-images.githubusercontent.com/40268737/135668151-5643510b-e425-4c36-b89c-087df9986816.PNG">

That copy also needs updating. It should read:

```
To query all indices, use *.
```

The asterisk should be formatting to indicate it's text input.